### ISSUE TITLE
Designing a poster for How to nail poster presentation 

### ISSUE THREAD

**Issue by gnurenga** [pythonindia/inpycon2019-tasks#105]
Reference:
https://healthpsychtam.com/2017/12/14/how-to-nail-an-academic-poster-presentation/

**Comment by bravegnu** [comment]
The most important question to answer today, is "Why present a poster?" rather than "How to present a poster?".

**Comment by abhishekmishragithub** [comment]
> The most important question to answer today, is "Why present a poster?" rather than "How to present a poster?".

@bravegnu  Have look here https://people.clas.ufl.edu/oppenhe/2014/06/09/7-reasons-why-you-should-present-posters-at-a-conference/ 
Above portal has nice reason on "Why to present a poster ?"

**Comment by abhishekmishragithub** [comment]
How we proceed now ?

> > The most important question to answer today, is "Why present a poster?" rather than "How to present a poster?".
> 
> @bravegnu Have look here https://people.clas.ufl.edu/oppenhe/2014/06/09/7-reasons-why-you-should-present-posters-at-a-conference/
> Above portal has nice reason on "Why to present a poster ?"

How we proceed now? @bravegnu @gnurenga

**Comment by bravegnu** [comment]
We have to customize it for PyCon India, and post it to our blog and our attendees.

**Comment by abhishekmishragithub** [comment]
> We have to customize it for PyCon India, and post it to our blog and our attendees.

@bravegnu , @gnurenga  Please give feedback :

Why should you go to all the trouble to present a poster at PyCon India? there are many good reasons to do so.

1. Posters offer the opportunity to engage with PyCon attendees interested in the same subject 
and application as yourself. 
You might strike up a conversation with someone about your poster that may lead to future collaboration or even a job offer! One-on-one conversations about your poster enable you to talk about very specific aspects of your research that may not be possible in a presentation.

2. A poster presentation is a good way to disseminate your work, particularly if your research falls within 
a narrow field of specialization. It also gives you the opportunity to actively engage with others during 
the conference in a way that networking alone does not, since by its very nature the poster defines the 
topic of conversation.

3. If your topic does not fit within the conference themes for the tracks, a poster is a good way to present research that doesn’t fit into one of the track categories well. That’s not to say your research doesn’t fit the conference or the audience, it just may not be a good fit for the agenda. 
Certainly posters can offer a strong supplement to the conference agenda, as well as provide you with a 
means to present your research.

4. Another good reason to present a poster is that you can expect to receive compliments on your work 
and critiques that can help you in the future. An insightful, impartial discussion of your work by a peer can often provide ideas for improving your work downstream.

5. Sometimes a poster session is better than an oral presentation due to time constraints in the latter. 
During a typical oral presentation you will have 5 minutes for questions and comments while a poster 
session typically lasts several hours, allowing for more in-depth discussions. Moreover, not all people will 
manage to make it to your oral presentation because there will probably be concurrent tracks.

6. Poster sessions offer the opportunity to practice your presentation skills as well. 
One advantage of presenting a poster is that it enables you to try explaining a concept in different ways, 
and see which explanation is most well received. At the same time that you’re honing your ability to 
communicate verbally, you can also hone your ability to present information visually in a simple way.

7. A poster session is ideal for the early stages of the research, when you may not have much more than an idea, and you stand to benefit a lot from discussing your idea with other researchers & PyCon attendee's from the same field.

In short, poster presentations provide value, both for attendees who discuss your poster content with you, and for yourself because of the opportunities highlighted above.

**Comment by gnurenga** [comment]
@abhishekmishragithub  looks good to me

**Comment by abhishekmishragithub** [comment]
@bravegnu have look to points now, and let me know how to proceed.

Why should you go to all the trouble to present a poster at PyCon India? there are many good reasons to do so.

1. Posters offer the opportunity to engage with PyCon attendees interested in the same subject 
and application as yourself. 
You might strike up a conversation with someone about your poster that may lead to future collaboration or even a job offer! One-on-one conversations about your poster enable you to talk about very specific aspects of your project/topic that may not be possible in a presentation.

2. A poster presentation is a good way to disseminate your work, particularly if your topic or project falls within a narrow field of specialization. It also gives you the opportunity to actively engage with others during the conference in a way that networking alone does not, since by its very nature the poster defines the topic of conversation.

3. If your topic does not fit within the conference themes for the tracks, a poster is a good way to present your topic that doesn’t fit into one of the track categories well. That’s not to say your topic doesn’t fit the conference or the audience, it just may not be a good fit for the agenda. 
Certainly posters can offer a strong supplement to the conference agenda, as well as provide you with a 
means to present your work.

4. Another good reason to present a poster is that you can expect to receive compliments on your work 
and critiques that can help you in the future. An insightful, impartial discussion of your work by a peer can often provide ideas for improving your work downstream.

5. Sometimes a poster session is better than an oral presentation due to time constraints in the latter. 
During a typical oral presentation you will have 5 minutes for questions and comments while a poster 
session typically lasts several hours, allowing for more in-depth discussions. Moreover, not all people will manage to make it to your oral presentation because there will probably be concurrent tracks.

6. Poster sessions offer the opportunity to practice your presentation skills as well. 
One advantage of presenting a poster is that it enables you to try explaining a concept in different ways, and see which explanation is most well received. At the same time that you’re honing your ability to communicate verbally, you can also hone your ability to present information visually in a simple way.

7. A poster session is ideal for the early stages of the project/work, when you may not have much more than an idea, and you stand to benefit a lot from discussing your idea with other researchers & PyCon attendee's from the same field.

In short, poster presentations provide value, both for attendees who discuss your poster content with you, and for yourself because of the opportunities highlighted above.

**Comment by bravegnu** [comment]
Looks much better now.

**Comment by bravegnu** [comment]
Here are few good reasons to present a poster at PyCon India:

1. Posters offer the opportunity to engage with attendees interested in the same subject and application as yourself. You might strike up a conversation with someone about your poster that may lead to future collaborations or even a job offer!

2. If you are new to conferences, or if you are intimidated about doing a talk in front of huge audience, posters are for you. Posters are a great way to test the waters, and can be stepping stone for your next talk at a conference.

3. The entire conference attendees is available during the poster session, as there are no other tracks scheduled in parallel. This makes it better than the talk sessions, where there are multiple concurrent tracks, and attendees can only attend one of them.

4. A poster presentation is a good way to disseminate your work, particularly if your topic or project falls within a narrow field of specialization. It also gives you the opportunity to actively engage with others during the conference in a way that networking alone does not, since by its very nature the poster defines the topic of conversation.

5. Another good reason to present a poster is that you can expect to receive compliments on your work and critiques that can help you in the future. An insightful, impartial discussion of your work by a peer can often provide ideas for improving your work downstream.

In short, poster presentations provide value, both for attendees who discuss your poster content with you, and for yourself because of the opportunities highlighted above. If this has got you interested, head to https://in.pycon.org/cfp/posters-2019/proposals/ for more details and to submit a poster proposal.

Credits: Partly based on https://people.clas.ufl.edu/oppenhe/2014/06/09/7-reasons-why-you-should-present-posters-at-a-conference/

**Comment by rajkumartreads** [comment]
I made a draft poster for the said content. Added titles to the points. I see there is scope to improve the content. Feedback pls.

![why_poster](https://user-images.githubusercontent.com/4427534/61104032-5634eb80-a492-11e9-80fb-8a953b4d45d1.png)

**Comment by bravegnu** [comment]
I would prefer, a different set of sub-topics, than the ones mentioned above.

  1. Engage with Attendees
  2. Test the Waters
  3. No Parallel Sessions
  4. Specialized Topics
  5. Get Feedback

**Comment by rajkumartreads** [comment]
Draft 2

![why_poster_2](https://user-images.githubusercontent.com/4427534/61104639-bdec3600-a494-11e9-9468-c9bc26e0819e.png)

**Comment by rajkumartreads** [comment]
Draft 3. Updated last date.

![why_poster_3](https://user-images.githubusercontent.com/4427534/61348429-40526c80-a87e-11e9-9a47-2a577284e603.jpg)
